README.md

```diff
@@ -1,17 +1,53 @@
 # cilium-sysdump
 
-# Overview
- The cilium-sysdump tool collects all information in a cluster required to
- troubleshoot issues with Cilium and Hubble.
+The cilium-sysdump tool collects all information in a cluster required to troubleshoot issues with Cilium and Hubble.
 
-# Prerequisites
-- Requires Python >= 2.7.*
-- Requires `kubectl`. 
-- `kubectl` should be pointing to your cluster before running the tool.
+## Prerequisites
+- Python >= 2.7
+- kubectl
+- pip (required only if you intend to rebuild the zip file)
```

Comment:
I would omit this as it might confuse users. `pip` is only required for dev purposes.